src/main/java/frc/robot/commands/ArmMoveToFloorCommand.java

```diff
@@ -24,8 +24,9 @@ public void initialize() {
     @Override
     public void execute() {
         double power = pidController.calculate(armSystem.getAbsEncoderPositionDegrees(), RobotMap.ARM_FLOOR_ANGLE);
+        pidController.setIZone(RobotMap.ARM_PID_I_ZONE);
```

Comment:
don't change Izone during calculation, set it in the ctor once.

Reply:
done